index.html

```diff
@@ -1,12 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="theme-color" content="#e30062" />
 
     <title>Welcome to Real Dev Squad</title>
 
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Real Dev Squad is a community of developers, where we come together to learn, use that in building projects, prepare for interviews, help each-other, put those projects online and try to help the set of new developers or freshers, helping them get better jobs and higher salaries with quality work."
+    />
+    <meta name="keywords" content="RDS, developers,Real Dev Squad" />
```

Comment:
I came across some blogs/ docs which state that the meta tag `keywords` is not currently used as it is not considered for page ranking and has no SEO value as such. For reference -> https://developers.google.com/search/blog/2009/09/google-does-not-use-keywords-meta-tag

should will still consider adding it? what are your thoughts on it?